noir_stdlib/src/hash/sha256.nr

```diff
@@ -110,8 +110,10 @@ pub fn sha256_var<let N: u32>(msg: [u8; N], message_size: u64) -> [u8; 32] {
 
         // If the block is filled, compress it.
         // An un-filled block is handled after this loop.
-        if msg_byte_ptr == BLOCK_SIZE {
-            h = sha256_compression(msg_u8_to_u32(msg_block), h);
+        if msg_start < message_size {
+            if msg_byte_ptr == BLOCK_SIZE {
```

Comment:
```suggestion
        if msg_start < message_size & msg_byte_ptr == BLOCK_SIZE {
```
Should be able to collapse this if statement

Reply:
Ah yea, had error with this, but just had to add brackets for each condition
Updated now